pkg/api/config_test.go

```diff
@@ -32,7 +32,38 @@ func TestValidate(t *testing.T) {
 				},
 			},
 			expectedValid: false,
-			expectedError: "Test should not be called 'images' because it gets confused with '[images]' target",
+			expectedError: "test should not be called 'images' because it gets confused with '[images]' target",
+		},
+		{
+			id: "BuildRoot",
```

Comment:
You introduce four unique error conditions -- test for all of them, as well as the correct case